### PR TITLE
Hjalte fixing some stuff.

### PR DESCRIPTION
**Pull request by Hjaltesorgenfrei**

### CODE REVIEW

File: report/01_Introduction.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 
-The course intended to teach how to maintain and evolve a legacy system.
+Throughout the course students were expected to learn how to maintain and evolve a legacy system.
 For this purpose, a legacy twitter clone, implemented in Flask with Python 2 running on an old macOS version, was given to the group along with the task of refactoring the application.
-The goal was for it to run on a modern operating system, Ubuntu 18.04, and not use outdated and unsupported technologies. To facilitate a smoother development and minimize lead time, DevOps practices like Continuous Integration (CI) and Continuous Deployment (CD) were to be utilized. In addition to CI/CD, extensive logging and monitoring were to be employed to aid in spotting and fixing problems.
+The refactoring goal was for it to run on a modern operating system, Ubuntu 18.04, and not use outdated and unsupported technologies. To facilitate a smoother development and minimize lead time, DevOps practices like Continuous Integration (CI) and Continuous Deployment (CD) were to be utilized. In addition to CI/CD, extensive logging and monitoring were to be employed to aid in spotting and fixing problems.
```
Comment (Jakobis) [inline report/01_Introduction.md]: "The goal of the refactoring"

File: report/11_Systems-Design.md [hunk 2]
```diff
@@ -1,9 +1,10 @@
 ### System Design 
+
 Our version of the MiniTwit application is a web app written in C# using ASP.NET. Data is stored using Entity Framework Core (EF Core) as a database abstraction layer working with a PostgreSQL database. This stack was chosen because it has both excellent support, maintenance, and documentation. Furthermore, this means that many third-party packages are available, like SeriLog which makes interaction with ElasticSearch painless and Prometheus-net for plug and play communication with Prometheus.
 
 The application has three main C# Projects.
 
-- MiniTwit.Web.App, which contains the web app that users interact with.
+- MiniTwit.Web.App, which contains the web app and api that users interact with.
```
Comment (Jakobis) [inline report/11_Systems-Design.md]: I think we are inconsistent about whether it's api or API, we should probably make a decision about it

File: report/11_Systems-Design.md [hunk 3]
```diff
@@ -12,10 +13,10 @@ The application also has other projects containing utility functions, as well as
 ![Component diagram](./images/component_diagram.png)<br/>
 *Figure 1: A component diagram describing the main components of the MiniTwit system.*
 
-As illustrated in figure 1, the `MiniTwitContext` allows the repositories to access the data relevant to them. The repositories then contain the appropriate methods for passing the data on in specific formats to the frontend controllers, contained in `MiniTwit.Web.App`. The `HomeController` passes this data on to the views in the Views directory, where they are used when accessed in a browser, while the `ApiController` instead exposes the API that the simulator uses.
+As illustrated in figure 1, the `MiniTwitContext` allows the repositories to access the data relevant to them. The repositories then contain the appropriate methods for passing the data on in specific formats to the frontend controllers, contained in `MiniTwit.Web.App`. The `HomeController` passes this data on to the views in the Views directory, which are used when accessing from a browser, while the `ApiController` instead exposes the API that the simulator uses.
```
Comment (Jakobis) [inline report/11_Systems-Design.md]: "which is" instead of "which are"

File: report/11_Systems-Design.md [hunk 4]
```diff
@@ -12,10 +13,10 @@ The application also has other projects containing utility functions, as well as
...
 
 The application is written completely in C# because the team wanted to learn more about the different aspects of using .NET.
-After all, it made it easier to connect all the parts of the application, and the team had experience with it, allowing work to be done faster.
+That team already had experience with it, made it easier to connect all the parts of the application, and allowed for work to be done faster.
```
Comment (Jakobis) [inline report/11_Systems-Design.md]: That team?

**Review by Jakobis — CHANGES_REQUESTED** [review]

File: report/11_Systems-Design.md (hunk 2 above)
Comment (tobloef) [reply report/11_Systems-Design.md]: Use API

**Review by Jakobis — APPROVED** [review]

**Pull request merged**